cpp/src/parquet/parquet.thrift

```diff
@@ -182,7 +182,7 @@ enum ConvertedType {
  * Representation of Schemas
  */
 enum FieldRepetitionType {
-  /** This field is required (can not be null) and each record has exactly 1 value. */
```

Comment:
We shouldn't modify the `parquet.thrift`, since it's from parquet-format repo: https://github.com/apache/parquet-format